projects/1.21/assets/modrinth-flashback/flashback/lang/zh_cn.json

```
@@ -0,0 +1,28 @@
+{
+  "flashback.select_replay": "选择回放",
+  "flashback.select_replay.open": "打开回放",
+  "flashback.select_replay.loading_replays": "正在加载回放",
+  "flashback.select_replay.replay": "回放",
+  "flashback.select_replay.data_read": "读取回放数据中……",
+  "flashback.select_replay.delete_question": "你确定你要删除这个回放吗？",
+  "flashback.select_replay.duration": "时长：%s",
+  "flashback.option.per_second": "%s/秒",
+  "flashback.option.automatically_start": "自动开始",
+  "flashback.option.automatically_start.description": "加入世界/服务器时自动开始回放录制",
+  "flashback.option.automatically_finish": "自动结束",
+  "flashback.option.automatically_finish.description": "离开世界/服务器时自动完成回放录制",
+  "flashback.option.show_recording_toasts": "显示录制弹窗",
+  "flashback.option.show_recording_toasts.description": "开始/停止回放录制时在右上角显示录制弹窗",
+  "flashback.option.quicksave": "快速保存",
+  "flashback.option.quicksave.description": "使用默认值保存回放，而不会打开「保存回放」窗口",
```

```suggestion
  "flashback.option.quicksave.description": "使用默认值保存回放，而不会打开“保存回放”屏幕",
```
https://zh.minecraft.wiki/w/%E8%B0%83%E8%AF%95%E5%B1%8F%E5%B9%95
我知道很奇怪

---

其他已解决的对话点一下resolve conversation

---

是窗口……